src/parser.jl

```diff
@@ -21,16 +21,18 @@ const RATIO_FILE_NAME = "ratios.txt"
 const VEHICLES_FILE_NAME = "vehicles.txt"
 
 # An Instance structure that is used to format data as we want.
-struct Instances
```

Comment:
I think you should rename your `INSTANCES` to `Instances`, a full capital word is used for constants.